integrations/monday-com/src/index.ts

```
@@ -0,0 +1,177 @@
+        registered: [],
+      },
+    })
+    const { registered } = stateResponse.state.payload
+    const newWebhooks: Webhook[] = []
+
+    for (const boardId of ctx.configuration.boardIds) {
+      for (const webhookName of webhooks) {
+        if (!registered.find((webhook) => webhook.name === webhookName && webhook.boardId === boardId)) {
+          const result = await monday.createWebhook(webhookName, webhookUrl, boardId)
+
+          newWebhooks.push({
+            name: webhookName,
+            boardId,
+            webhookId: result.create_webhook.id,
+          })
+        }
+      }
+    }
+
+    if (newWebhooks.length > 0) {
+      await client.setState({
+        id: ctx.integrationId,
+        type: 'integration',
+        name: 'webhooks',
+        payload: {
+          registered: [...registered, ...newWebhooks],
+        },
+      })
+    }
+  },
+  unregister: async ({ client, ctx, logger }) => {
+    const vanillaClient = getVanillaClient(client)
+
+    await vanillaClient.deleteTable({
+      table: 'MondayItemsTable',
+    })
+
+    const monday = MondayClient.create({
+      personalAccessToken: ctx.configuration.personalAccessToken,
+    })
+
+    try {
+      const stateResponse = await client.getState({
+        id: ctx.integrationId,
+        type: 'integration',
+        name: 'webhooks',
+      })
+      const registered = stateResponse.state.payload.registered
+
+      for (const webhook of registered) {
+        logger.info('delete webhook', {
+          boardId: webhook.boardId,
+          webhookId: webhook.webhookId,
+          name: webhook.name,
+        })
+        await monday.deleteWebhook(webhook.webhookId).catch((err) => {
+          logger.forBot().error(JSON.stringify(err, null, 2))
+          logger.forBot().error('Could not delete Monday.com webhook with ID ', webhook.webhookId)
+        })
+      }
+    } catch {
+      // TODO should we just let uninstallation continue or is it beneficial to raise an error here?
```

Yes, it is fine to just swallow the error in this specific case